### ISSUE TITLE
Document rave.overrides

### ISSUE THREAD

**Issue by KidkArolis** [RaveJS/rave#41]
In the top level package.json/bower.json:

```
"rave": {
    "overrides": {
        "foo": {
            "main": "./some/other/main.js"
        }
    }
}
```

I could write this up, but what would be the best place for it?

`README.md`
`docs/overrides.md`
`docs/rave.md`

**Comment by unscriptable** [comment]
One of the reasons it's not documented is that I'm not confident that the current implementation is the best way. I guess it's good enough for now?  If anyone has opinion, please speak up. :)

My current feelings about docs is that we need an overall explanation about how users should extend and customize a rave-based application. `overrides` would fit into that nicely. 

On the other hand, there's no need to hold you up. It'll be easy to refactor the docs when I get around to writing the overall explanation. 

tl;dr `docs/overrides.md` is fine for now.

**Comment by unscriptable** [comment]
Added as of 0.3.0.
